examples/tutorials/advanced/date_time_charts.py

```diff
@@ -285,7 +285,7 @@
 # Starting off with ``WS``, adding this string means that only
 # Western/Left (**W**) and Southern/Bottom (**S**) borders of
 # the plot will be shown. For more information on this, please
-# refer to :doc:`frame instructions </tutorials/frames>`.
+# refer to :doc:`frame instructions </tutorials/beginner/frames>`.
 #
 # The other important item in the ``frame`` list is
 # ``"sxa1Of1D"``. This string modifies the secondary
```

Comment:
Not really related, but could you update L14 of this `date_time_charts.py` file to get a proper thumbnail? Right now it's showing up like this:

![image](https://user-images.githubusercontent.com/23487320/140603731-f0e5117c-cf85-4350-9147-5f8f47d992ed.png)